packages/sdk/src/sdk/api/notifications/NotificationsApi.ts

```diff
@@ -0,0 +1,59 @@
+import type { Configuration } from '../../api/generated/default'
+import type { AuthService, EntityManagerService } from '../../services'
+import { Action, EntityType } from '../../services/EntityManager/types'
+import { parseParams } from '../../utils/parseParams'
+
+import {
+  MarkAllNotificationsAsViewedRequest,
+  UpdatePlaylistLastViewedAtRequest,
+  MarkAllNotificationsAsViewedSchema,
+  UpdatePlaylistLastViewedAtSchema
+} from './types'
+
+export class NotificationsApi {
```

Comment:
Curious why this is standalone? Where do the getters live?

Reply:
only in full atm lol